pkg/apis/extensions/types.go

```diff
@@ -638,3 +638,39 @@ type ClusterAutoscalerList struct {
 
 	Items []ClusterAutoscaler `json:"items"`
 }
+
+// A pod selector is a label query over a set of pods. The result of matchLabels and
+// matchExpressions are ANDed. An empty podSelector matches all objects. A null
```

Comment:
I would just use "pod selector" in these later 2 cases, as well, to match the first sentence.